src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -6,110 +6,38 @@
 
   <stache-page-summary>
     <p>
-      During local development with SKY UX, local files are loaded into SKY UX Host. To load your local single-page application into SKY UX Host, you must install the <a href="https://raw.githubusercontent.com/blackbaud/skyux-builder/master/ssl/skyux-ca.crt">skyux-ca.crt</a> root certificate. Follow the installation instructions appropriate for your machine.
+      For local development with SKY UX, you must install the self-signed certificate that enables your SPA to load local files into SKY UX Host.
     </p>
   </stache-page-summary>
 
   <stache-page-anchor>
     Automated installation for Linux, Mac, and Windows
   </stache-page-anchor>
-
+  
+  <p>
+    To add the certificate to your machine's list of trusted certificates and validate it, run the following command:
+  </p>
   <sky-code-block>
-    skyux certs trust
+    skyux certs install
   </sky-code-block>
-
-  <p>To learn about other options, see the <a routerLink="/learn/reference/cli-commands/certs"><sky-code>skyux certs</sky-code> documentation</a>.</p>
-
-  <stache-page-anchor>
-    Manual installation for Mac
-  </stache-page-anchor>
-
-  <ol>
-    <li>
-      <sky-code-block>
-        skyux certs generate
-      </sky-code-block>
-    </li>
-    <li>
-      <p>
-        Open the Keychain Access application.
-      </p>
-    </li>
-    <li>
-      <p>
-        Under the Keychains list on the left, select <strong>login</strong>.
-      </p>
-    </li>
-    <li>
-      <p>
-        Under the Category list, select <strong>Certificates</strong>.
-      </p>
-    </li>
-    <li>
-      <p>
-        Open the <sky-code>.skyux/certs</sky-code> folder in your home directory and drag <sky-code></sky-code> into the list of certificates.
-      </p>
-    </li>
-    <li>
-      <p>
-        Double-click the new <strong>SKY UX</strong> item in the list to open the certificate's info window.
-      </p>
-    </li>
-    <li>
-      <p>
-        Expand the Trust section near the top of the info window, and select the <strong>Always Trust</strong> option for <strong>Secure Sockets Layer (SSL)</strong>.
-      </p>
-    </li>
-    <li>
-      <p>
-        Close the info window.
-      </p>
-    </li>
-  </ol>
+  <p>
+    The certificate is required to securely run <sky-code>skyux serve</sky-code>, <sky-code>skyux e2e</sky-code>, and <sky-code>skyux build --serve</sky-code>. For more information, see the <a routerLink="/learn/reference/cli-commands/certs"><sky-code>skyux certs</sky-code> documentation</a>.
+  </p>
 
   <stache-page-anchor>
-    Manual installation for iOS Simulator on Mac
+    Additional steps for iOS Simulator on Mac
   </stache-page-anchor>
 
   <ol>
-    <li>
```

Comment:
In regards to the iOS simulator, all of these are still valid, only the very first `skyux certs generate` step should be removed.

Reply:
I added the steps back in. Let me know if they look OK now.